### ISSUE TITLE
Inserted, hidden video tracks become audio tracks after re-opening project

### ISSUE THREAD

**Issue by ochilan** [mltframework/shotcut#625]
Hello,

there is a particular series of actions that degrade a video track to an audio track after re-opening a project.

Repro:
- Start with a new project.
- Add two video tracks.
- Insert a new video track (e.g. between the two tracks, i.e. right click the bottom track -> Insert Track).
- Hide the newly inserted video track (click the eye icon).
- Save and re-open the project.

Observed:
- The inserted, hidden video track now appears to be an audio track.

The problem does not seem to happen when using added (not inserted) video tracks, and also not when the inserted track is not hidden before saving.

Shotcut version 18.08.14

**Comment by ddennedy** [comment]
I believe you because I had to repair someone's project file, and because once out of about 15 tries I did reproduce it. However, I am unable to reproduce it again in order to debug this further. Are you able to repeat it? If so, how exactly are you executing Insert Track because it all depends on the current track, which has the gold background, and when you right-click a track head the track is both made current and the track head selected - at this point nothing is gold because the tracks are empty.

**Comment by ochilan** [comment]
I made a video showing the repro (it has two parts – first using only the contextual menu to add and insert tracks, and then using Ctrl-I to add the first two tracks, with the same result). I also added two mlt files, one resulting in adding three tracks, and one adding two tracks and then inserting a third track. If you take a look at the diff, it seems like the inserted track is missing a "shotcut:video" property, and a transition has an additional "disable" property.

Note that if you open the "add_add_insert.mlt", it does not appear to be broken since I didn't hide the second track using the eye icon. If I open it, hide the track, save and re-open it, then it is broken, i.e. the video track becomes an audio track.

Hope this helps.

[shotcut_bug.zip](https://github.com/mltframework/shotcut/files/2327407/shotcut_bug.zip)
